### ISSUE TITLE
Reproducibility of Paper Results

### ISSUE THREAD

**Issue by marja-w** [Wluper/Retrograph#17]
Hello,

I have been trying to reproduce the results that you have presented in the paper, but my pre-trained and fine-tuned adapters only achieve an accuracy of about 0.6 on the GLUE dataset (using 4_pretrain_adapter.py, glue_1_download_glue.sh, glue_2_finetune_adapter.sh). 

Since the random walks are generated using a probability, could you share the exact random walks that you used in your experiment? Maybe the bad performance is caused by the difference in training data.
